.github/workflows/release.yml

```diff
@@ -31,7 +31,9 @@ jobs:
           IFS=""
           echo "$PGP_PRIVATE_KEY" | gpg --dearmor > $HOME/secring.gpg
           echo "$PGP_PASSPHRASE" > $HOME/passphrase.txt
-          echo "$PGP_PUBLIC_KEY" > $HOME/pubring.gpg
+          # Unfortunately, the public key must be in binary format because sigstore does not support ASCII-armored keys.
+          # $PGP_PUBLIC_KEY is base64-encoded, so we decode it to binary and write it to $HOME/pubring.gpg.
+          echo "$PGP_PUBLIC_KEY_BINARY" | base64 --decode > $HOME/pubring.gpg
```

Comment:
GitHub won't let me annotate the proper location since it's outside the diff, but this job's `env.PGP_PUBLIC_KEY` has to be renamed.